themes/f5-2/layouts/404.html

```diff
@@ -0,0 +1,27 @@
+{{ partial "header.html" . }}
+
+<main role="main">
+    <article class="article" itemscope itemtype="http://schema.org/Article">
+        <header data-interchange="[{{ "/images/404-small.jpg" | absURL }}, small], [{{ "/images/404-medium.jpg" | absURL }}, medium], [{{ "/images/404-large.jpg" | absURL }}, large]" title="{{ .Title }}" alt="{{ .Title }}">
```

Comment:
data-interchange is a foundation feature and should be removed. 

That said, using the header images is a miss on all pages.  Play with them